src/Livewire/DataTables/TicketList.php

```diff
@@ -19,6 +31,10 @@ class TicketList extends DataTable
         'created_at',
     ];
 
+    public array $columnLabels = [
+        'ticket_type.name' => 'Ticket Type',
```

Comment:
i think you dont need this with the new data table version, would be more consistant with ticket type -> name in the label